nginx/frontend.conf

```diff
@@ -1,44 +1,66 @@
-upstream frontend {
-  server localhost:3000;
-  server localhost:9000 backup;
+server {
+  listen 443 ssl;
+  server_name preview.local.dev-gutools.co.uk;
+
+  location / {
+    proxy_http_version 1.1;
+    proxy_pass http://localhost:9000;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header Host $http_host;
+    proxy_redirect default;
+    proxy_buffering off;
+  }
+
+  ssl_certificate     preview.local.dev-gutools.co.uk.crt;
+  ssl_certificate_key preview.local.dev-gutools.co.uk.key;
```

Comment:
How are these keys generated?

Reply:
by [`dev-nginx`](https://github.com/guardian/dev-nginx), see changes to `nginx/setup.sh`